.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -1,5 +1,6 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default" />
```

Comment:
This might mean you are not using the Project Default?

Please verify your settings are like this 
![image](https://user-images.githubusercontent.com/3358282/79971934-06f20180-848d-11ea-99d0-90ec5c89d504.png)

Reply:
I have been using project default for inspection. Also, these inspections are working for me as studio now points out several inspection issues which it didn't do earlier. 

<img width="1018" alt="Screen Shot 2020-04-22 at 9 45 04 PM" src="https://user-images.githubusercontent.com/3069373/80060100-8bbb3a80-84e2-11ea-892d-627dc605f73c.png">

Reply:
My apologies, I screenshot the wrong thing

![image](https://user-images.githubusercontent.com/3358282/80078085-50eaee00-8546-11ea-8040-a47883601685.png)

Reply:
That too is set to use project settings. 

<img width="1016" alt="Screen Shot 2020-04-23 at 1 44 21 AM" src="https://user-images.githubusercontent.com/3069373/80078605-fe88dd80-8503-11ea-8195-714d1f0ffd0d.png">

Everytime i switch branch I keep stashing this change. Is there a way to fix it?

Reply:
![image](https://user-images.githubusercontent.com/3358282/80079906-ca83db80-8548-11ea-8076-2e7e5133f93b.png)

This is the change that happens when I set my code style to `default`

If you just discard this change does it come back? You can try switching codestyles a bit (and clicking apply) and see what happens

Reply:
I found a potential [solution](https://youtrack.jetbrains.com/issue/IDEA-207303#focus=streamItem-27-3299338.0-0) that may work?